pom.xml

```diff
@@ -48,16 +48,16 @@
          https://github.com/confluentinc/common (see common's pom.xml for kafka.version).
     -->
     <properties>
-        <algebird.version>0.13.4</algebird.version>
+        <algebird.version>0.13.7</algebird.version>
         <avro.version>1.9.1</avro.version>
-        <chill.version>0.9.2</chill.version>
+        <chill.version>0.9.5</chill.version>
         <docker.skip-build>false</docker.skip-build>
         <docker.skip-test>false</docker.skip-test>
         <java.version>1.8</java.version>
         <licenses.version>6.0.0-SNAPSHOT</licenses.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <scala.version>${kafka.scala.version}.8</scala.version>
-        <scalatest.version>3.0.5</scalatest.version>
+        <scala.version>${kafka.scala.version}.2</scala.version>
```

Comment:
There's something silly about using a variable for the major and minor, while plugging in a literal patch number. Can we just expand the reference? Note, having done so in the past would have avoided the issue that necessitated this PR to begin with.

Also note, an alternative (and maybe preferable) fix for this issue is to set the scala version back to what it was before.

Reply:
I can expand the scala.version to be hard-coded, but note that I cannot easily keep us at scala 12 since other dependencies we have already moved to scala 13 in their master branch.

One risk is that hard-coding `<scala.version>` while keeping other dependencies with `{kafka.scala.version}` could still cause compilation issues in the future.

Reply:
Thanks @guozhangwang , at risk of defending a silly idea...

Of course we would still get failures in the future, since we are targeting snapshot dependencies of other Confluent repos, but I'd put forth that at least those failures would be sane. As opposed to a failure produced by concatenating two unrelated strings and using the result as the version of Scala to build against, which seems _extremely_ subtle. It just happens that you were able to track it down because 2.13.8 doesn't exist at all, but next time Kafka bumps the scala version, what if 2.14.2 does happen to exist? It'll be exceptionally tricky to spot that we've silently gone from 2.13 to 2.14, since nothing at all would have changed in the pom. It would just start failing one day.

Reply:
Yup. I've updated the PR to hard-code the scala version and also not inheriting the kafka.scala.version but set it our own.